frame/staking/Cargo.toml

```diff
@@ -17,7 +17,6 @@ static_assertions = "1.1.0"
 serde = { version = "1.0.101", optional = true }
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = ["derive"] }
 sp-std = { version = "3.0.0", default-features = false, path = "../../primitives/std" }
-# TWO_PHASE_NOTE:: ideally we should be able to get rid of this.
```

Comment:
We didnt get rid of anything?

Reply:
We still need to generate the static solution type here at compile time `sp_npos_election::generate_compact_solution!`

Reply:
and will be fixed here: https://github.com/paritytech/substrate/pull/8368